flash/core/data/process.py

```diff
@@ -413,15 +400,6 @@ def per_batch_transform_on_device(self, batch: Any) -> Any:
         """
         return self.current_transform(batch)
 
-    def available_data_sources(self) -> Sequence[str]:
-        """Get the list of available data source names for use with this
-        :class:`~flash.core.data.process.Preprocess`.
-
-        Returns:
-            The list of data source names.
-        """
-        return list(self._data_sources.keys())
-
     def data_source_of_name(self, data_source_name: str) -> DataSource:
```

Comment:
This function still needs to go, a Registry should help with that. We'd probably need to add a deprecation notice though.